src/components/Drawer.js

```diff
@@ -0,0 +1,146 @@
+const React = require('react');
+const Velocity = require('velocity-animate');
+
+const StyleConstants = require('../constants/Style');
+
+const Icon = require('../components/Icon');
+
+const Drawer = React.createClass({
+  propTypes: {
+    duration: React.PropTypes.number,
+    easing: React.PropTypes.array,
+    isOpen: React.PropTypes.bool,
```

Comment:
@bsbeeks @cerinman the component is not controlling its open, close state -- it receives an isOpen flag as a prop on mount. To make this more clear, I'm removing the default prop of isOpen and setting isOpen to isRequired.